sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -17,6 +17,8 @@ public enum ServiceVersion
     public partial class DataLakeDirectoryClient : Azure.Storage.Files.DataLake.DataLakePathClient
     {
         protected DataLakeDirectoryClient() { }
+        public DataLakeDirectoryClient(string connectionString, string fileSystemName, string path) { }
+        public DataLakeDirectoryClient(string connectionString, string fileSystemName, string path, Azure.Storage.Files.DataLake.DataLakeClientOptions options) { }
```

Comment:
is there a reason we can't just have optional options ?

Reply:
This is a good question, I'm not actually sure what the answer is, but all other other constructors are organized this way.